cmd/configschema/go.mod

```diff
@@ -8,9 +8,9 @@ require (
 	github.com/open-telemetry/opentelemetry-collector-contrib v0.75.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/redisreceiver v0.75.0
 	github.com/stretchr/testify v1.8.2
-	go.opentelemetry.io/collector v0.75.1-0.20230424163446-8e33ded10872
-	go.opentelemetry.io/collector/component v0.75.1-0.20230424163446-8e33ded10872
-	go.opentelemetry.io/collector/receiver/otlpreceiver v0.75.1-0.20230424163446-8e33ded10872
+	go.opentelemetry.io/collector v0.76.0
```

Comment:
this version is surprising to me. i ran it locally and the collector was upgraded to 

```
go: upgraded go.opentelemetry.io/collector v0.75.1-0.20230424163446-8e33ded10872 => v0.76.1-0.20230426191218-56daa378f504
```